articles/machine-learning/how-to-configure-auto-train.md

```diff
@@ -339,9 +339,12 @@ No&nbsp;criteria | If you do not define any exit parameters the experiment conti
 `max_concurrent_trials`| Represents the maximum number of trials (children jobs) that would be executed in parallel. It's a good practice to match this number with the number of nodes your cluster
 
 ## Run experiment
-> [!WARNING]
+> [!NOTE]
 > If you run an experiment with the same configuration settings and primary metric multiple times, you'll likely see variation in each experiments final metrics score and generated models. The algorithms automated ML employs have inherent randomness that can cause slight variation in the models output by the experiment and the recommended model's final metrics score, like accuracy. You'll likely also see results with the same model name, but different hyperparameters used. 
 
+> [!WARNING]
+> If you have set rules in firewall and/or Network Security Group over your workspace, verify that required permissions are given to inbound and outbound network traffic as defined in [Configure inbound and outbound network traffic](how-to-access-azureml-behind-firewall.md).
```

Comment:
How will vision customers discover this piece of information?
Also, since the warning is at AutoML level, please add relevant stakeholders from other teams (NLP, Forecasting etc).